plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/alg/LayoutProcessorConfiguration.java

```diff
@@ -0,0 +1,240 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Kiel University - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.elk.core.alg;
+
+import java.util.Set;
+
+import com.google.common.collect.Sets;
+
+/**
+ * Instances of this class specify {@link ILayoutProcessor layout processors} that should be executed in the different
+ * processing slots before, in between, and after the phases of a layout algorithm. Layout processor configurations are
+ * typically specified by {@link ILayoutPhase layout phases} to specify their processor dependencies. The information
+ * can be used by an {@link AlgorithmAssembler} to assemble the layout algorithm. Each processor is only ever added once
+ * to each processing slot.
```

Comment:
Once to each processing slot. It can be added to different processing slots. Seems that the documentation is easy to misinterpret here, I'll extend it a bit to cover this point.